caching/invalidation.py

```diff
@@ -37,22 +39,22 @@
 
 def make_key(k, with_locale=True):
     """Generate the full key for ``k``, with a prefix."""
-    key = encoding.smart_str('%s:%s' % (CACHE_PREFIX, k))
+    key = encoding.smart_text('%s:%s' % (CACHE_PREFIX, k))
     if with_locale:
-        key += encoding.smart_str(translation.get_language())
+        key += encoding.smart_text(translation.get_language())
```

Comment:
I think both of these instances should be `smart_bytes` because apparently `k` is expected to be a sequence of bytes at this point. https://docs.djangoproject.com/en/1.8/topics/python3/#string-handling